modules/web/server/src/main/resources/logback.xml

```diff
@@ -16,9 +16,9 @@
     <then>
         <!-- Files are rotated daily for up to 90 days -->
         <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-            <file>/gemsoft/var/log/seqexec-ng/seqexec-ng.log</file>
+            <file>/home/software/logs/seqexec-ng.log</file>
             <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-                <fileNamePattern>/gemsoft/var/log/seqexec-ng/seqexec-ng.%d{yyyy-MM-dd}.log</fileNamePattern>
+                <fileNamePattern>/home/softawre/logs/seqexec-ng.%d{yyyy-MM-dd}.log</fileNamePattern>
```

Comment:
This and line 19 should not be merged into main

Reply:
Solved